### PR TITLE
Add ONNX py3 CI cases

### PR DESCRIPTION
**Pull request by houseroad**
So far, we only have py2 ci for onnx. I think py3 support is important. And we have the plan to add onnxruntime backend tests, which only supports py3.

### CODE REVIEW

File: .circleci/cimodel/data/caffe2_build_data.py [hunk 1]
```diff
@@ -63,6 +65,7 @@ def is_build_only(self):
         ] or self.find_prop("distro_version").name == "macos"
 
 
+
```
Comment (bddppq) [inline .circleci/cimodel/data/caffe2_build_data.py]: remove this

File: .circleci/cimodel/data/caffe2_build_definitions.py [hunk 2]
```diff
@@ -30,6 +30,7 @@ class Conf:
     def get_cudnn_insertion(self):
 
         omit = self.language == "onnx_py2" \
```
Comment (bddppq) [inline .circleci/cimodel/data/caffe2_build_definitions.py]: this (`onnx_py2`) can be removed now?

File: .circleci/cimodel/data/caffe2_build_data.py [hunk 3]
```diff
@@ -22,7 +22,7 @@
         (Ver("gcc", "5"), [XImportant("onnx_py2")]),
```
Comment (bddppq) [inline .circleci/cimodel/data/caffe2_build_data.py]: this can be removed now?

**Review by bddppq — COMMENTED** [review]
good start :-)

File: .circleci/cimodel/data/caffe2_build_data.py (hunk 1 above)
Comment (houseroad) [reply .circleci/cimodel/data/caffe2_build_data.py]: sure

File: .circleci/cimodel/data/caffe2_build_definitions.py (hunk 2 above)
Comment (houseroad) [reply .circleci/cimodel/data/caffe2_build_definitions.py]: I would suggest to keep py2 one until we retire py2 internally. py2 helped us caught several bugs which were not manifested in py3.

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+import unittest
+import sys
+import onnxruntime  # noqa
+
+
+class TestONNXRuntime(unittest.TestCase):
+
+    def test_onnxruntime_installed(self):
+        self.assertTrue('onnxruntime' in sys.modules)
```
Comment (bddppq) [inline test/onnx/test_pytorch_onnx_onnxruntime.py]: I assume this will be removed once meaningful tests are added in the future?

**Review by bddppq — APPROVED** [review]
LGTM

File: test/onnx/test_pytorch_onnx_onnxruntime.py (hunk 4 above)
Comment (houseroad) [reply test/onnx/test_pytorch_onnx_onnxruntime.py]: absolutely!

**Pull request closed without merging**